private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+interface State {}
+
+interface Props {}
+
+class Dashboard extends React.Component<Props, State> {
+	render() {
+		return <React.Fragment>DASHBOARD</React.Fragment>;
+	}
+}
+
+export default Dashboard;
```

Comment:
As per my comment above, using a HOC would look something like:

`export default requireAuthentication(Dashboard)`

This will allow the logic to be easily changed in future

Example: https://riptutorial.com/reactjs/example/30238/higher-order-component-that-checks-for-authentication